src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue

```diff
@@ -110,20 +96,41 @@
       },
     },
     computed: {
-      ...mapState('places-auto-complete', {placeloading: 'isFindPending'}),
-      ...mapState('geocode', {geoloading: 'isFindPending'}),
-      ...mapGetters('places-auto-complete', {addresses: 'list'}),
+      ...mapState(usePlacesAutoComplete, {placeloading: 'isFindPending'}),
+      ...mapState(useGeocode, {geoloading: 'isFindPending'}),
+      ...mapState(usePlacesAutoComplete, {addresses: 'items'}),
       loading() {
         return this.placeloading || this.geoloading;
       },
+      attrs() {
+        let newVal = {...this.$attrs};
+        // attrs defaults
+        this.$lset(newVal, 'attrs.filled', this.$lget(newVal, 'attrs.filled', true));
+        this.$lset(newVal, 'attrs.clearable', this.$lget(newVal, 'attrs.clearable', true));
+        this.$lset(newVal, 'attrs.clear-icon', this.$lget(newVal, 'attrs.clear-icon', 'close'));
+        this.$lset(newVal, 'attrs.use-input', this.$lget(newVal, 'attrs.use-input', true));
+        this.$lset(newVal, 'attrs.hide-selected', this.$lget(newVal, 'attrs.hide-selected', true));
+        this.$lset(newVal, 'attrs.fill-input', this.$lget(newVal, 'attrs.fill-input', true));
+        this.$lset(newVal, 'attrs.input-debounce', this.$lget(newVal, 'attrs.input-debounce', '0'));
+        this.$lset(newVal, 'attrs.placeholder', this.$lget(newVal, 'attrs.placeholder', 'Enter your address (ex. 123 N Main St, City, ZZ 12345)...'));
+        this.$lset(newVal, 'attrs.option-label', this.$lget(newVal, 'attrs.option-label', 'description'));
+
+        // div-attrs defaults
+        this.$lset(newVal, 'div-attrs.class', this.$lget(newVal, 'div-attrs.class', 'col-12 col-sm-6'));
+        return newVal;
+      },
     },
     methods: {
-      ...mapMutations('places-auto-complete', {clearAddresses: 'clearAll'}),
-      ...mapActions('places-auto-complete', {findAddresses: 'find'}),
-      ...mapActions('geocode', {findgeocoded: 'find'}),
-      async clearInput() {
+      // ...mapMutations('places-auto-complete', {clearAddresses: 'clearAll'}),
```

Comment:
There is a `clearAll` action in the state!

Reply:
resolve